files/zh-tw/web/http/basics_of_http/identifying_resources_on_the_web/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Identifying resources on the Web
+slug: Web/HTTP/Basics_of_HTTP/Identifying_resources_on_the_Web
+---
+
+{{HTTPSidebar}}
+
+HTTP 請求的目標被稱為 「資源」，其性質並未進一步定義；它可以是一個文件、一張照片或其他任何東西。每個資源都由統一資源標識符（{{Glossary("URI")}}）識別，該 URI 在整個 HTTP 中用於識別資源。
+
+## URLs 和 URNs
+
+### URLs
+
+最常見的 URI 形式是統一資源定位符（URL），也稱為 _網址_。
+
+```url
+https://developer.mozilla.org
+https://developer.mozilla.org/en-US/docs/Learn/
+https://developer.mozilla.org/en-US/search?q=URL
+```
+
+可以將任何一個 URLs 輸入到瀏覽器的地址欄中，以告訴它加載相關的網頁（資源）。
+
+URL 由不同的部分組成，一些是必需的，而其他是可選的。一個更複雜的例子可能如下所示：
+
+```url
+http://www.example.com:80/path/to/myfile.html?key1=value1&key2=value2#SomewhereInTheDocument
+```
+
+### URNs
+
+統一資源名稱（URN）是一個在特定命名空間中以名稱識別資源的 URI。
+
+```url
+urn:isbn:9780141036144
+urn:ietf:rfc:7230
+```
+
+這兩個 URN 分別對應到：
+
+- 《0984》一書，作者是喬治·奧威爾（George Orwell）。
+- IETF 規範 7230，標題為「超文本傳輸協議（HTTP/1.1）：消息語法和路由」。
+
+## 統一資源識別符（URIs）的語法
+
+### 方案(Scheme) 或 協議(protocol)
+
+- ![Protocol](mdn-url-protocol@x2.png)
+  - `http://` 是個協議，它指示瀏覽器應該使用哪種協議。通常情況下，這是 HTTP 協議或其安全版本 HTTPS。網頁需要使用這兩者之一，但瀏覽器也知道如何處理其他協議，比如 `mailto:`（用於打開郵件客戶端）或 `ftp:`（用於處理文件傳輸），所以如果你看到這些協議，也不要感到驚訝。一些常見的方案包括：

```

Comment:
https://zh.wikipedia.org/zh-tw/%E7%BD%91%E7%BB%9C%E4%BC%A0%E8%BE%93%E5%8D%8F%E8%AE%AE

```suggestion
### 方案或協定

- ![協定](mdn-url-protocol@x2.png)
  - `http://` 是個協定（protocol），它指示瀏覽器應該使用哪種協定。通常情況下，這是 HTTP 協議或其安全版本 HTTPS。網頁需要使用這兩者之一，但瀏覽器也知道如何處理其他協定，比如 `mailto:`（用於打開郵件客戶端）或 `ftp:`（用於處理文件傳輸），所以如果你看到這些協定，也不要感到驚訝。一些常見的方案（scheme）包括：
```